docs/guide/taxii.ipynb

```diff
@@ -2406,7 +159,6 @@
     "indicators = tc_source.query([f1])\n",
```

Comment:
This is minor, but at least when I tested with medallion's test data, this didn't return multiple objects, but using a filter of `!=` did.

Reply:
No it doesnt. I was just pretending that it did (or would have).  Originally, when querying Brets server, we would get 126 indicators in return. And since he only had indicators, the example was just pretending that there was a mix of objects but we were only getting indicators back, when in reality you only got all the indicators every time (bc was broken).

The example is still correct, it just might look off as its iterating over the list and there is only one.